docs/workflow/ci/ci_health.md

```diff
@@ -0,0 +1,167 @@
+# CI Health and Investigation
```

Comment:
Can you please rename the file to something like coreclr-ci-health.md so that it's clear what this is about?